parsers/global-libs.ts

```diff
@@ -1,9 +1,24 @@
 import _ from 'lodash';
 import tinycolor from 'tinycolor2';
 import got from 'got';
+import { AllowedFormat } from '../types/tokens';
 
-export default {
+const Libs = {
   _,
   tinycolor,
   got,
+  // SpServices: SpServices,
```

Comment:
Remove if not used

Reply:
It's necessary to inform user that SpServices is allow in `libs` but it's not set here?